spec/slashdeploy/service_spec.rb

```diff
@@ -90,13 +90,28 @@
   end
 
   describe '#unlock_environment' do
```

Comment:
This test seemed to suffer from a copy paste error and wasn't actually testing what we thought it was testing. I think this change fixes it, or at least actually tests another "code path"